src/aiodynamo/credentials.py

```diff
@@ -174,7 +175,7 @@ def __init__(
             return
 
         try:
-            profile = parser[profile_name]
+            profile = parser[str(profile_name)]
```

Comment:
is this str cast only for mypy, or is there some other reason behind it?

Reply:
Simply for mypy